pom.xml

```diff
@@ -133,6 +133,11 @@
             <version>4.5.10</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
```

Comment:
It won't let me comment on it above, but is easymock a runtime dependency or a test dependency?

Reply:
can you change the easymock dependency scope to test